app/src/debug/java/pl/gov/mc/protego/ui/base/BaseActivity.kt

```diff
@@ -0,0 +1,22 @@
+package pl.gov.mc.protego.ui.base
+
+import android.os.Bundle
+import kotlinx.android.synthetic.main.toolbar.*
+
+abstract class BaseActivity : BaseVariantActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setupToolbar()
```

Comment:
Nie pokazuje konfliktu, ale na masterze to zostało przeniesione do `setContent()`:
https://github.com/ProteGO-app/android/blob/master/app/src/main/kotlin/pl/gov/mc/protego/ui/base/BaseActivity.kt#L19